Cargo.toml

```diff
@@ -63,6 +63,7 @@ expect-test = "1.5.0"
 rustversion = "1.0"
 test-log = { version = "0.2.11", features = ["trace"] }
 trybuild = "1.0"
+get-size2 = { version = "0.4.1", features = ["derive"] }
```

Comment:
Can we remove the dependency and use a manual `size` implementation in tests instead?